source/includes/_clients.md

```diff
@@ -30,6 +30,19 @@ curl -X POST \
  https://tools.brightlocal.com/seo-tools/api/v1/clients-and-locations/clients/
 ```
 
+### Creating a client
+
+```csharp
+api request = new api("<INSERT_API_KEY>", "<INSERT_API_SECRET>");
+
+var parameters = new api.Parameters();
+       parameters.Add("name", "Le Bernardin");
+       parameters.Add("company-url", "http://www.example.com");
```

Comment:
Might as well put Le Bernardin's actual address as you've put their name. Generally I think real examples that will work are better than dummy data.